webpack.config.js

```diff
@@ -84,7 +88,25 @@ if (isProd) {
     }),
     new webpack.DefinePlugin({
       'process.env': { NODE_ENV: JSON.stringify(env) }
-    })
+    }),
```

Comment:
I prefer arrows since they autobind this context to the function, and it can avoid weird error cases. I don't care that much though, just keep this context in mind since javascript this is different than other things in regards to function context.